ops/deploy.dev.sh

```diff
@@ -154,7 +154,7 @@ services:
       ETH_MNEMONIC: $eth_mnemonic
       ETH_NETWORK_ID: $eth_network_id
       ETH_RPC_URL: $eth_rpc_url
-      HUB_WALLET_ADDRESS: $hub_wallet_address
+      HOT_WALLET_ADDRESS: $hot_wallet_address
```

Comment:
@bohendo any reason this was named like this and changed in prod entry?

Reply:
I've been preferring `hub_wallet_address` since hot wallet isn't very descriptive. (Whose hot wallet? Mine? Yours? The hub's?). idc that much though.